scripts/install_local.py

```diff
@@ -0,0 +1,122 @@
+"""
+Installs aqueduct from the local repo. No aqueduct server can be running.
+
+After this script runs, `aqueduct server` will start the backend with the local changes.
+The sdk will also be updated with any local changes.
+"""
+
+import os
+import platform
+import random
+import string
+import subprocess
+import sys
+
+base_directory = os.path.join(os.environ["HOME"], ".aqueduct")
+server_directory = os.path.join(os.environ["HOME"], ".aqueduct", "server")
+
+def execute_command(args, cwd=None):
+    with subprocess.Popen(args, stdout=sys.stdout, stderr=sys.stderr, cwd=cwd) as proc:
+        proc.communicate()
+        if proc.returncode != 0:
+            raise Exception("Error executing command: %s" % args)
+
+
+def update_executable_permissions():
+    execute_command(["chmod", "755", os.path.join(server_directory, "bin", "server")])
+    execute_command(["chmod", "755", os.path.join(server_directory, "bin", "executor")])
+    execute_command(["chmod", "755", os.path.join(server_directory, "bin", "migrator")])
+
+
+def update_config_yaml(file):
+    s = string.ascii_uppercase+string.digits
+    encryption_key = ''.join(random.sample(s,32))
+    api_key = ''.join(random.sample(s,32))
+
+    with open(file, "r") as sources:
+        lines = sources.readlines()
+    with open(file, "w") as sources:
+        for line in lines:
+            if "<BASE_PATH>" in line:
+                sources.write(line.replace("<BASE_PATH>", server_directory))
+            elif "<ENCRYPTION_KEY>" in line:
+                sources.write(line.replace("<ENCRYPTION_KEY>", encryption_key))
+            elif "<API_KEY>" in line:
+                sources.write(line.replace("<API_KEY>", api_key))
+            else:
+                sources.write(line)
+
+
+if __name__ == "__main__":
+    print("Current directory should be the root directory of the aqueduct repo.")
+    cwd = os.getcwd()
+    if not cwd.endswith("aqueduct"):
+        print("Your working directory is %s" % cwd)
+        exit(1)
+
+    # Create the ~/.aqueduct directory is it does not already exist. Copied from `/src/python/bin/aqueduct`.
```

Comment:
This and some of the helpers in this script are copied from the CLI. I'm not sure if I should devote time right now trying to extract out these common helpers.